openwisp_controller/connection/connectors/ssh.py

```diff
@@ -133,17 +133,17 @@ def exec_command(
         # store command exit status
         exit_status = stdout.channel.recv_exit_status()
         # log standard output
-        output = stdout.read().decode('utf8').strip()
+        output = stdout.read().decode('utf-8', 'ignore')
```

Comment:
nice, does this fix your issue? What does `ignore` do? I was not aware of it, maybe we adding a comment above this line with a reference to the docs may be helpful to our future selves or other maintainers

Reply:
Yes, it fixes the issue. I didn't use `ignore` myself in the past, but I read through the docs and found it to be a suitable way to handle the decoding errors. The output looks sane again.